docs/extreme_fregrid_sample_runscript.txt

```diff
@@ -1,17 +1,19 @@
 ﻿# The NCTools app fregrid_parallel is the parallel version of fregrid, and
 # it is particularly useful for processing with large grids. fregrid will
-# generate this message when grids are too large for it :
+# generate this message when the remapping weights file is not available and
+# grids are too large for it to process:
```

Comment:
Is it possible to change the error message from fregrid to disambiguate what nxgrid really means.  We all know it refers to the number of exchange grid cells exceeding some hard-set limit.

Reply:
@bensonr 
I updated the message to this:

FATAL Error: The xgrid size is too large for resources.
 nxgrid is greater than MAXXGRID/nthreads; increase MAXXGRID,
 decrease nthreads, or increase number of MPI ranks.

The new related docs were also update, as your suggestions and those of @ceblanton.